main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -316,11 +334,35 @@ object Completion {
   /**
     * Represents a keyword completion.
     *
-    * @param name the name of the keyword.
-    * @param priority the completion priority of the keyword.
+    * @param name      the name of the keyword.
+    * @param priority  the completion priority of the keyword.
     */
   case class KeywordCompletion(name: String, priority: Priority) extends Completion
 
+  /**
+    * Represents a keyword literal completion (i.e. `true`)
+    *
+    * @param name      the name of the keyword.
+    * @param priority  the priority of the keyword.
+    */
+  case class KeywordLiteralCompletion(name: String, priority: Priority) extends Completion
+
+  /**
+    * Represents a completion for a keyword with call syntax (i.e. `checked_cast`)
+    * 
+    * @param name      the name of the keyword.
+    * @param priority  the priority of the keyword.
+    */
+  case class KeywordCallCompletion(name: String, priority: Priority) extends Completion
+
+  /**
+    * Represents a collection keyword completion (i.e. `Hash#{}`)
```

Comment:
Please write the actual things we support :)

Reply:
Oh, my bad. I guess I wrote that comment went a bit too fast. ^^'. I'll fix that to `List#{}` or something. That comment i left is complete nonsense ^^'